pbcommand/services/cli.py

```diff
@@ -115,17 +115,18 @@ def dataset_walker(root_dir):
     return walker(root_dir, filter_func)
 
 
-def import_dataset(sal, dataset_meta_type, path):
-    log.info("Mock {s} importing dataset type {t} path:{p}".format(t=dataset_meta_type, p=path, s=sal))
-    # FIXME
-    return True
+def import_dataset(sal, path):
+    dataset_meta_type = is_dataset(path)
+    log.info("{s} importing dataset type {t} path:{p}".format(
+        t=dataset_meta_type, p=path, s=sal))
+
+    return sal.run_import_dataset_by_type(dataset_meta_type, path)
 
 
 def import_datasets(sal, root_dir):
     for path in dataset_walker(root_dir):
-        meta_type = ""
-        result = import_dataset(sal, meta_type, path)
-        yield result
+        result = import_dataset(sal, path)
+        # yield result
```

Comment:
The plural version does not return anything. Since the singular version used to return True, maybe it returns a boolean now? If so, maybe you want the conjunction? Or maybe the singular version shouldn't return anything either? Doesn't look like a bug, but consistency can be helpful.